neurom/fst/tests/test_neuritefunc.py

```diff
@@ -213,6 +213,12 @@ def test_section_branch_orders():
                     (0, 1, 1,  # type 3, basal dendrite
                      0, 1, 1)) # type 2, axon
 
+def test_section_strahler_orders():
+    strahler_orders = list(_nf.section_strahler_orders(SIMPLE))
+    assert_allclose(strahler_orders,
+                    (2, 1, 1,  # type 3, basal dendrite
```

Comment:
Thank you for the test! There seem to be something wrong with this figure in Wiki or I do not understand the definition :/ Could we meet tomorrow to check this (I'm not in Geneva today)?